Add documentation around OIDC_OP_LOGOUT_URL_METHOD. Fixes #217.

johnpaulett opened the pull request:
Fixes #217

johngian commented:
Hi @johnpaulett thanks for the PR. Since we changed the function signature to take the ``request`` object can you also a ``Backwards-incompatible`` change in the ``HISTORY.rst``.

At the moment if somebody has already implemented the ``OIDC_OP_LOGOUT_URL_METHOD`` this [1] would break their setup.

[1] https://github.com/mozilla/mozilla-django-oidc/commit/c7f79029cde83bf7cbbf270d3e61c778275c6f1b

johnpaulett commented:
@johngian makes sense. Modified commit and force-pushed to branch. Added to HISTORY.rst and did a `..versionchanged` in the settings.rst.  Assumed version number was going to be 0.7.0, since backwards incompatible change, but obviously, feel free to change or let me know what version you want.

johngian commented:
Thanks @johnpaulett for the contribution :rocket:
